300-CNAB-security.md

```diff
@@ -0,0 +1,163 @@
+# Cloud Native Application Bundles Security (CNAB-Sec) 1.0 WD
+
+The CNAB Security specification augments the [CNAB Core specification](100-CNAB.md) by standardizing on security mechanisms for signing, verifying, and attesting CNAB packages. It describes both a client/registry security model and a verification chain (provenance) model. By design, this provides security mechanisms that work in air-gapped networks.
+
+This specification is distinct from the CNAB Core specification. An implementation may comply with the CNAB Core specification, and yet not comply with this specification. The use of terms such as MUST and SHOULD in this document are statements about how a CNAB implementation may fulfill the CNAB Security specification _only_.
+
+> An earlier version of the CNAB Core specification used OpenPGP-based mechanism to sign and verify bundles. This specification supersedes that portion of the specification.
+
+This specification addresses two related problems:
+
+- Securing the delivery of packages from a source to a client
+- Ensuring the provenance of a bundle robustly, even across networks and over long periods of time
+
+## Key Terms
+
+In addition to the terminology introduced in the CNAB Core specification, this specification uses additional terms:
+
+- *Acceptance Criteria:* The conditions that must obtain before a particular CNAB may be considered acceptable for use. This may be, in some cases, as simple as "the CNAB bundle has the expected digest" or as complicated as a tree-structured diagram of attestations.
+- *Attestation:* an assertion, cryptographically verifiable, that a particular identity has performed a particular task. For example, an attestation may capture the assertion that "Developer Padma has approved version 1.2.3 of the WebApp bundle."
+- *Bundle:* This term refers to a CNAB package, and is used to clarify that the object in question is a particular package, not the CNAB specification.
+- *CNAB Client:* Any CNAB-aware program that operates on a package generated elsewhere.
+- *CNAB Registry:* Any repository of CNAB bundles. While there is a specification for how such bundles should be stored (the CNAB Registry specification), this specification assumes only that a registry is a remote storage system from whence CNAB bundles can be pulled and (optionally) to which they may be pushed.
```

Comment:
nit: link to the registry specification?

Reply:
Good catch! Fixing.